content/en/integrations/_index.md

```diff
@@ -1,6 +1,5 @@
 ---
-title: "Integrations"
-weight: 50
```

Comment:
This weight is useless since the page doesn't appear in any (top- or side-) nav.